app/lib/pre_assembly/assembly_directory.rb

```diff
@@ -4,20 +4,22 @@ module PreAssembly
   # Represents the assembly structure on the filesystem,
   # Used by PreAssembly::DigitalObject
   class AssemblyDirectory
-    def self.create(druid_id:)
-      new(druid_id: druid_id).tap(&:create_object_directories)
+    def self.create(druid_id:, base_path: '')
```

Comment:
Wondering about why the default is `''`. Are there cases in which this kwarg might be omitted?

Reply:
you are correct: unnecessary and removed

however, in working on a PR for #1109, I know have a separate question for andrew that may affect this work, so putting a [HOLD] on it for now